src/machine.rs

```diff
@@ -918,16 +918,16 @@ impl OlmMachine {
         let me = self.inner.clone();
 
         future_to_promise(async move {
-            let request = me.backup_machine().backup().await?;
-
-            let Some(request) = request else {
-                return Ok(None);
-            };
+            match me.backup_machine().backup().await? {
```

Comment:
I preferred the `let else` over the `match` :stuck_out_tongue:.

YMMV, happy for you to change it.

Reply:
I went with the `match` to format this similarly to `OlmMachine.get_missing_sessions`, which is what gave me the idea of working with a `(request_id, request)` tuple in the first place.  Thanks for letting me keep it :slightly_smiling_face: